quickstarts/aws/aws-storage-gateway/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: aws-storage-gateway
+description: |-
+  ## What is AWS Storage Gateway?
+  AWS Storage Gateway is a set of hybrid cloud storage services that provide on-premises access to virtually unlimited cloud storage.
+  ### Get started!
+  Start monitoring AWS Storage Gateway by connecting Amazon Web Services (AWS) to New Relic! Take a look at our documentation for information on how to set up this integration.
+  ### More info
+  Check out the [documentation](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-storage-gateway-integration/) to learn more about New Relic monitoring for AWS Storage Gateway.
+summary: |-
+  Monitor AWS Storage Gateway by connecting AWS to New Relic
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: AWS Storage Gateway
+documentation:
+  - name: AWS Storage Gateway installation docs
+    description: |
+      Monitor AWS Storage Gateway by connecting AWS to New Relic.
+    url: >-
```

Comment:
This is not working. I'm assuming this is the link to the docs that will go live later today?

Reply:
Yes @brammerl , docs is not yet live. I have asked the team to merge it today. Thats the reason in PR I have not checked the docs live checkbox.